### DISCUSSION TITLE
Qual a diferença entre trabalhar numa startup x multinacional?

### DISCUSSION THREAD

**Discussion by ReneSena** [frontendbr/forum#2052] [Discussão]
Fala, pessoal! Tudo bem?

Gostaria saber de vocês, sobre as experiências de trabalho em relação a startup x multinacional?  Tendo em visto a questões como: Ambiente de trabalho, autonomia, crescimento e também os afazeres.  Acho que essa é uma duvida de muitos.

Obs: Eu vi que em multinacional, geralmente o dev não é tão focado em código, sim mais na parte de gestão do Ti, não sei se isso  é verdade.

**Comment by amandagdev** [comment]
Trabalhei em 2 startups e agora to em uma multi e pra >mim< baseado na minha experiência não tem nem comparação. estou muito melhor e mais feliz, não tem aquela pressão ridicula de startup claro que não é perfeito mas é bem melhor. Mas acho que é diferente de startup "consolidadas" pq ai muda né? assim como existe multis muito engessadas e que pode ser ruim também. E esse ponto que você falou sobre não ser focado em codigo nunca vi, que eu saiba você vai trabalhar com código normalmente mas entender do negocio é essencial.
